multirun/def.bzl

```diff
@@ -21,7 +36,8 @@ def _multirun_impl(ctx):
         default_runfiles = info.default_runfiles
         if default_runfiles != None:
             transitive_depsets.append(default_runfiles.files)
-        content.append("echo Running %s\n./%s\n" % (shell.quote(str(command.label)), shell.quote(exe.short_path)))
+        full_command = "%s ./%s %s" % (envs, shell.quote(exe.short_path), attrs)
```

Comment:
Environment variables may contain special characters that cannot be just put into the script directly. I'm not sure how to properly escape it. Maybe `shell.quote()` is enough.

Reply:
I addressed that comment now in a simple way - tested it and it works for our internal use-case. Let me know what you think.

This right now only works if arguments are passed in via `--a=b` but not `--a b` to keep the logic simple. Also happy to try and support the second case. Also I tried using `shell.quote` without splitting the args/envs but that did not work.